### PR TITLE
Add buff_geo and other small changes

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by richardellison** [comment]
Looks good Robin. Maybe use message() instead of print() so it works with suppressMessages()?

File: NEWS [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+* Functionality for reading in the UK's stats19 data: `read_stats19_*`
+  functions download, unzip and re-categorise the data.
+
+* `read_table` functions added for reading Australian OD data
+  (@richardellison).
+
+* `decode_gl` added to decode Google polylines (@richardellison).
+  Towards osrm routing in R!
+
```
Comment (Robinlovelace) [inline NEWS]: Any more new features to add @richardellison? I imagine - good to communicate this ready for the next release to CRAN.

File: R/toptail.R [hunk 2]
```diff
@@ -12,23 +17,27 @@
...
+    # Create buffer for geographic or projected crs
+    if(!is.projected(l)){
+      sel <- buff_geo(lpoints, width = toptail_dist, ...)
+    } else {
+      sel <- rgeos::gBuffer(lpoints, width = toptail_dist, ...)
+    }
+
     if(rgeos::gContainsProperly(sel, l1)){
-      print(paste0("Line ", i, " is completely removed by the clip and",
+      message(paste0("Line ", i, " is completely removed by the clip and",
```
Comment (Robinlovelace) [inline R/toptail.R]: That should fix it.

File: NEWS (hunk 1 above)
Comment (richardellison) [inline NEWS]: I'm about to put in a pull request with a function for reading in GTFS data.

**Pull request merged**